tracer/build/_build/Build.cs

```diff
@@ -160,6 +170,20 @@ void DeleteReparsePoints(string path)
         .DependsOn(BuildNativeLoader)
         .DependsOn(CreateRootDescriptorsFile);
 
+    Target BuildTracerHomeForAwsLambda => _ => _
+        .Description("Builds the native and managed src, and publishes the tracer home directory for AWS Lambda")
+        .Requires(() => IsLinux)
+        .After(Clean)
+        .DependsOn(CreateRequiredDirectories)
+        .DependsOn(Restore)
+        .DependsOn(PublishManagedTracerForAwsLambda)
+        .DependsOn(CompileTracerNativeSrcLinux) // CompileNativeSrc
+        .DependsOn(PublishNativeTracerUnix)     // PublishNativeTracer
+        //.DependsOn(DownloadLibDdwaf)
+        //.DependsOn(CopyLibDdwaf)
+        .DependsOn(CompileNativeLoaderLinux)    // BuildNativeLoader -> CompileNativeLoader
+        .DependsOn(PublishNativeLoaderUnix);    // BuildNativeLoader -> PublishNativeLoader
```

Comment:
Prefer the non-Linux specific versions, as that takes of everything for you. Any reason you didn't do that here?
```suggestion
        .DependsOn(CompileNativeSrc)
        .DependsOn(PublishNativeTracer)
        //.DependsOn(DownloadLibDdwaf)
        //.DependsOn(CopyLibDdwaf)
        .DependsOn(CompileNativeLoader)
        .DependsOn(PublishNativeLoader);
```

Reply:
It seemed redundant since it's _always_ Linux when building the AWS Lambda layer. I reverted like you asked here, but then I ended up with Lambda-specific versions of `PublishNativeTracer` and `PublishNativeLoader`, so we'll see.